agu_oss/clean_data.py

```diff
@@ -12,11 +12,9 @@ def open_and_clean(url):
     ----------
     url : string
         Path to the data  from the Berkeley earth  website
-
     Returns
```

Comment:
nice set of changes @aatish-5 !! no need for  additional line breaks in this file!